core/src/main/scala/org/json4s/reflect/ScalaType.scala

```diff
@@ -89,7 +89,7 @@ object ScalaType {
   /* end optimization */
 }
 
-class ScalaType(private val manifest: Manifest[_]) extends Equals {
+class ScalaType(private[json4s] val manifest: Manifest[_]) extends Equals {
```

Comment:
This would actually have to be public. What's the reason for it being private in the first place?

Reply:
I don't know what the original intention is but I guess the `manifest` has been used only inside of this class. I don't see any blockers that prevent changing its visibility.

`private[json4s]` is not good for the purpose here. It should be just public. If we go with `private[json4s]`, you have to place your serializers under `org.json4s` package.